client/main.lua

```diff
@@ -20,16 +20,32 @@ function DrawText3Ds(x, y, z, text)
     ClearDrawOrigin()
 end
 
+local function RaceCountDown()
+    ShowCountDown = true
+    while RaceCount ~= 0 do
+        FreezeEntityPosition(GetVehiclePedIsIn(PlayerPedId(), true), true)
+        PlaySound(-1, "slow", "SHORT_PLAYER_SWITCH_SOUND_SET", 0, 0, 1)
+        QBCore.Functions.Notify(RaceCount, 'primary', 800)
+        Wait(1000)
+        RaceCount = RaceCount - 1
+    end
+    ShowCountDown = false
+    RaceCount = 5
+    FreezeEntityPosition(GetVehiclePedIsIn(PlayerPedId(), true), false)
+    QBCore.Functions.Notify("GOOOOOOOOO!!!")
+end
+
 CreateThread(function()
     while true do
-        Wait(7)
+        local sleep = 1000
         if Races ~= nil then
             -- No race yet
             local pos = GetEntityCoords(PlayerPedId(), true)
             if RaceId == 0 then
                 for k in pairs(Races) do
                     if Races[k] ~= nil then
                         if #(pos - vector3(Races[k].startx, Races[k].starty, Races[k].startz)) < 15.0 and not Races[k].started then
+                            sleep = 1
```

Comment:
These 1s have to all be 0, you don't wait 1 millisecond to let it loop each frame

Reply:
Values have been updated.